velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -23,8 +23,19 @@ namespace facebook::velox::window::test {
 
 namespace {
 
-class NthValueTest : public WindowTestBase {
+const std::vector<std::string> kFrameClauses = {
```

Comment:
The frameClauses can be moved to WindowTestBase to re-use across multiple test files.

Reply:
Sure, thanks for the suggestion!